app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -432,6 +467,25 @@ const MeetingEditor = () => {
                     )}
                   </ConfirmModal>
                 )}
+                {isEditPage && canDelete && (
+                  <ConfirmModal
+                    title="Hente inn referat fra localstorage?"
+                    message={`Er du sikker på at du vil hente dette møtet? ${localStorage.getItem(
```

Comment:
If a confirm modal is to be used, it should inform that this action will overwrite the current report.